### PR TITLE
Remove unnecessary spec_helper requires in tests

### PR DESCRIPTION
**Pull request by kke**
Rspec adds spec/ to load path. You don't need any `require_relative "../../../spec_helper"` sorcery, 'require "spec_helper"` works fine.

### CODE REVIEW

**Comment by SpComb** [comment]
The `agent` includes a `.rspec` with `--require spec_helper`, so the agent specs don't need this require at all.

**Comment by kke** [comment]
I don't like including `.rspec` in git, because I you may want to make local changes to it.

**Comment by SpComb** [comment]
> I don't like including .rspec in git, because you may want to make local changes to it.

Hmm... well, the `server` and `agent` already include their `.rspec`... but the server doesn't have `--require spec_helper`, so it also has `require_relative '../../spec_helper'` in each spec :confounded:

Could we fix all three of the components to work the same way? My vote is for `.rspec` with `--require spec_helper`, but I'm also okay with `require 'spec_helper'`.

**Comment by kke** [comment]
I like to have `--profile --color --format d` in my cli `.rspec` :(

**Comment by SpComb** [comment]
sez the [rspec docs](http://www.rubydoc.info/github/rspec/rspec-core/RSpec/Core/Configuration)

> Configuration options are loaded from ~/.rspec, .rspec, .rspec-local, command line switches, and the SPEC_OPTS environment variable (listed in lowest to highest precedence; for example, an option in ~/.rspec can be overridden by an option in .rspec-local).

So we can stick `.rspec` into git, and you can use `.rspec-local`?

**Comment by kke** [comment]
Nice.

File: cli/.gitignore [hunk 1]
```diff
@@ -15,3 +15,4 @@ mkmf.log
 .idea
 *.gem
 /vendor/
+/.rspec-local
```
Comment (SpComb) [inline cli/.gitignore]: This could be in the top-level `.gitignore`

File: agent/.gitignore [hunk 2]
```diff
@@ -8,3 +8,4 @@ log/*.log
 /build/
 /release/
 /vendor/
+.rspec-local
```
Comment (SpComb) [inline agent/.gitignore]: Doesn't git pick it up from `../.gitignore`? I don't think this is needed here or in the cli/server.

**Pull request merged**